cmake/DAGMCConfig.cmake.in

```diff
@@ -0,0 +1,21 @@
+# - Config file for the Direct Accelerated Geometry Monte Carlo (DAGMC) package
+# It defines the following variables
+#  DAGMC_INCLUDE_DIRS - include directories for installed DAGMC headers
+#  DAGMC_LIBRARY_DIRS - location of installed DAGMC libraries
+#  DAGMC_LIBRARIES    - set of libraries installed with DAGMC, use to link applications against DAGMC
+
+# Declares the DAGMC package as found by CMake
+set(DAGMC_FOUND 1)
+
+set(DAGMC_MAJOR_VERSION @DAGMC_MAJOR_VERSION@)
+set(DAGMC_MINOR_VERSION @DAGMC_MINOR_VERSION@)
+set(DAGMC_PATCH_VERSION @DAGMC_PATCH_VERSION@)
+set(DAGMC_VERSION @DAGMC_VERSION@)
+ 
+message(STATUS "Configuring DAGMC ${DAGMC_VERSION}")
+
+set(DAGMC_INCLUDE_DIRS "@CMAKE_INSTALL_PREFIX@/@INSTALL_INCLUDE_DIR@" "@MOAB_INCLUDE_DIRS@")
+set(DAGMC_LIBRARY_DIRS "@CMAKE_INSTALL_PREFIX@/@INSTALL_LIB_DIR@" "@MOAB_LIBRARY_DIRS@")
+set(DAGMC_LIBRARIES "-ldagmc -ldagtally -lmakeWatertight -lpyne_dagmc @MOAB_LIBS@")
+# turn variable into list
+string(REGEX REPLACE " " ";" DAGMC_LIBRARIES "${DAGMC_LIBRARIES}")
```

Comment:
This might be nitpicky but lines 19-21 could be replaced with

`set(DAGMC_LIBRARIES dagmc dagtally makeWatertight pyne_dagmc @MOAB_LIBRARIES@)`

Reply:
On it
- [ ] Condense `DAGMC_LIBRARIES` definition

Reply:
And not using quotes makes CMake interpret a variable definition as a list